PythonTool/TagExpansion.py

```diff
@@ -25,20 +24,16 @@ def parse_arguments():
                         type=str,
                         help="Provide a path to the mesh file."
                         )
-    parser.add_argument("-o", "--outputfile",
-                        type=str,
-                        help="Provide a base name for the output files."
-                        )
 
     args = parser.parse_args()
 
     return args
 
 
-def get_vector_tags(mb, element):
+def get_tag_lists(mb, element):
     """
-    Create a list of each vector tag in the mesh file by identifying each tag on
-    a specific mesh element and determining if it is vector or scalar.
+    Create separate lists of each scalar and vector tag in the mesh file by
+    identifying each tag on a specific mesh element and determining the type.
```

Comment:
says it determines the type, is that the tag type? That doesn't get returned..

Reply:
That's an explanation of how it determines which list to put the tag into, not what the method actually does, sorry for the confusion!